web/frontend/src/components/utils/useChangeAction.tsx

```diff
@@ -16,6 +17,7 @@ const useChangeAction = (
   setShowModalError: (willShow: boolean) => void
 ) => {
   const { t } = useTranslation();
+  const { role } = useContext(AuthContext);
```

Comment:
In the rest of the code base it is called `authCtx`. Let's stick to that for consistency.